tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Services/SubscriptionsManagerTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Threading.Tasks;
+using Bot.Builder.Community.Adapters.Twitter.Webhooks.Services;
+using Microsoft.Extensions.Options;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace Bot.Builder.Community.Adapters.Twitter.Tests
```

Comment:
Maybe namespace should be `Bot.Builder.Community.Adapters.Twitter.Tests.Webhooks.Services`?